assembly/src/tokens/mod.rs

```diff
@@ -116,7 +116,12 @@ impl<'a> Token<'a> {
         assert_eq!(Self::USE, self.parts[0], "not a use");
         match self.num_parts() {
             0 => unreachable!(),
-            1 => Err(ParsingError::missing_param(self)),
+            1 => Err(
+                ParsingError::missing_param(
+                    self,
+                    "use <module_path_and_module_name>"
+                )
+            ),
```

Comment:
I think the correct format here would be: `use.<absolute_module_path>`

Reply:
Changed.